pointmatcher/DataPointsFiltersImpl.cpp

```diff
@@ -595,7 +596,47 @@ typename PointMatcher<T>::Vector DataPointsFiltersImpl<T>::SurfaceNormalDataPoin
 		}
 	}
 
-	return eigenVe.col(smallestId);
+  return eigenVe.col(smallestId);
+}
+
+template<typename T>
+Eigen::Matrix<T,3,1> DataPointsFiltersImpl<T>::SurfaceNormalDataPointsFilter::sortEigenValues(const Vector eigenVa) {
```

Comment:
Is there a way to generalize this function for any dimensions? Or at least 2D and 3D?